src/darwin/Framework/CHIP/MTRThreadOperationalDataset.h

```diff
@@ -49,9 +49,9 @@ extern size_t const MTRSizeThreadPSKc;
  */
 @property (nonatomic, nullable, copy, readonly) NSData * PSKc;
 /**
- *  The Thread network channel
+ *  The Thread network channel.  Always a 16-bit integer.
```

Comment:
nit: Does it worth mentioning that it is using an `unsigned` representation ?

Reply:
Yes, will do.